shell/platform/windows/flutter_windows_engine.cc

```diff
@@ -506,6 +506,21 @@ void FlutterWindowsEngine::ScheduleFrame() {
   embedder_api_.ScheduleFrame(engine_);
 }
 
+void FlutterWindowsEngine::SetNextFrameCallback(const fml::closure& callback) {
+  next_frame_callback_ = callback;
+
+  embedder_api_.SetNextFrameCallback(
+      engine_,
+      [](void* user_data) {
+        // Embedder callback runs on raster thread. Switch back to platform thread.
+        FlutterWindowsEngine* that =
```

Comment:
nit: We typically use `self` for these. We do use `that` (or `other`) pretty commonly, but almost always for operator= and operator== args. In this case, it's referring to this engine though.

Reply:
Fixed!